### PR TITLE
DeveloperGuide, AboutUs, ReadMe : Update Content

### PR DESCRIPTION
**Pull request by April0616**
update docs

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -783,25 +783,25 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
 
-|`* *` |user |access my address book using a password| ensure my data is secured
+|`* *` |user |access my address book using a password | ensure my data is secured
 
 |`* *` |user |have temporary tags for semester-based project groups |can delete my contacts after one semester
 
 |`* *` |user who likes different styles | change the theme(color) of the UI |
 
-|`*` |student with many friends |save my friends’ timetables |know when my friends are free to meet up
+|`*` |student with many friends |save my friends??? timetables |know when my friends are free to meet up
```
Comment (nbriannl) [inline docs/DeveloperGuide.adoc]: What is these `???`

**Review by nbriannl — DISMISSED** [review]
Change the ??? into '
But besides that all is good.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (April0616) [reply docs/DeveloperGuide.adoc]: ...some unrecognized ...symbols....sorry..I'll fix them

**Review by nbriannl — APPROVED** [review]
Thanks for the changes. All's good.

**Pull request merged**